beam/www/beam/pages/Demand.vue

```diff
@@ -36,10 +36,14 @@ useInfiniteScroll(
 
 		// TODO: move this to the server
 		data.forEach(row => {
-			row.count = { count: row.allocated_qty, of: row.total_required_qty }
-			row.label = `${row.item_code} from ${row.warehouse}`
+			row.count = { count: row.allocated_qty, of: `${row.total_required_qty}` }
```

Comment:
@lauty95 both the `count` and `of` values need to be a number. I'll look into adding type-hints for this file.